zone/client.h

```diff
@@ -1231,6 +1243,8 @@ class Client : public Mob
 	Object*						m_tradeskill_object;
 	PetInfo						m_petinfo; // current pet data, used while loading from and saving to DB
 	PetInfo						m_suspendedminion; // pet data for our suspended minion.
+	CharacterInstanceLockout	pending_instance_lockout;
```

Comment:
Not sure if we care about formatting, but the variable name doesn't line up with the other variable names above it

Reply:
Formatted the rest of the line (CTRL K + CTRL F)